csharp/ql/src/change-notes/2022-02-24-buildless-extractor-option.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
+---
+* The C# extractor now accepts an extractor option `buildless`, which is used to decide what type of extraction that should be performed. If `true` then buildless (standalone) extraction will be performed. Otherwise tracing extraction will be performed (default).
```

Comment:
Perhaps add: ``The option is added via `codeql database create --language=csharp -Obuildless=true ...` ``